<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalView</class>
 <widget class="QWidget" name="LocalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QProgressBar" name="bulk">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Принтер:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="printerName">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Идентификатор:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Тип слежения:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="sysId">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changeSysId">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Изменить</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="trackingType">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSpinBox" name="countAddP">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addPage">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Всего напечатано:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="totalPrinted">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Напечатано с последней заправки:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lastPrinted">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Добавить Страницы:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Средняя стоимость печати:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="price">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="showRefill">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Просмотреть заправки</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ЗАМЕНА КАРТРИДЖА</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 71, 71);</string>
           </property>
           <property name="text">
            <string>ПРОБЛЕМЫ С ПЕЧАТЬЮ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
